packages/orchestration/src/examples/stake-it/stake-it.flow.js

```diff
@@ -92,29 +92,23 @@ const makeAccountsHandler = async (
   });
 
   // @ts-expect-error LCA should have .monitorTransfers() method #9212
-  await heapVowE(localAccount).monitorTransfers(tap);
+  await localAccount.monitorTransfers(tap);
 
   const accountEntries = /** @type {[string, OrchestrationAccount<any>][]} */ ([
     ['agoric', localAccount],
     [chainName, stakingAccount],
   ]);
   const publicTopics = await Promise.all(
     accountEntries.map(async ([name, account]) => {
-      const { account: topicRecord } =
-        // @ts-expect-error getPublicTopics does not exist on OrchestrationAccountI
-        await heapVowE(account).getPublicTopics();
+      const { account: topicRecord } = await account.getPublicTopics();
       return /** @type {[string, PublicTopic<unknown>]} */ ([
         name,
         topicRecord,
       ]);
     }),
   );
   const portfolioHolder = makePortfolioHolder(accountEntries, publicTopics);
-  console.log('@@@portfolioHolder', portfolioHolder);
-  //  FIXME RangeError: Expected "promise" is same as "object".. from storagePath?
-  // const continuingOffer = portfolioHolder.asContinuingOffer();
-  // console.log('@@@continuingOffer', continuingOffer);
-  // return continuingOffer;
+  return portfolioHolder.asContinuingOffer();
```

Comment:
I think `asContinuingOffer` needs to be changed as well if you don't expect to [find a vow](https://github.com/Agoric/agoric-sdk/blob/bc48a8b2d76799c9c07a0888c496039c29049194/packages/orchestration/src/exos/cosmos-orchestration-account.js#L322C13-L322C30) for `publicSubscribers`.

Reply:
Thanks. I've pulled out these changes to https://github.com/Agoric/agoric-sdk/pull/9679 and tagged you as a reviwer